README.md

```diff
@@ -17,6 +17,13 @@
 [![CircleCI](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-r.svg?style=shield)](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-r) [![Join the chat at https://gitter.im/sonatype/nexus-developers](https://badges.gitter.im/sonatype/nexus-developers.svg)](https://gitter.im/sonatype/nexus-developers?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-r/depshield.svg)](https://depshield.github.io)
 [![Maven Central](https://img.shields.io/maven-central/v/org.sonatype.nexus.plugins/nexus-repository-r.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.sonatype.nexus.plugins%22%20AND%20a:%22nexus-repository-r%22)
 
+> **Huzzah!** R is now part of Nexus Repository Manager. Version 3.next includes the R plugin by default. The plugin source code is now in [nexus-public](https://github.com/sonatype/nexus-public) in [nexus-repository-r](https://github.com/sonatype/nexus-public/tree/master/plugins/nexus-repository-r)
+
+> **Filing issues:** If using 3.next or later please file any issues at https://issues.sonatype.org/.
```

Comment:
This implies that if you are using an older version of this plugin that you should file an issue somewhere else.

How about

```
Filing issues: Upgrade to the latest version of Nexus Repository Manager 3, to get the latest fixes and improvements, before filing any issues or feature requests at https://issues.sonatype.org/.
```

Reply:
If we're not going to disable issues here, you may also say don't use issues here :)

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-r/pull/139/commits/15e957ddc6f9601b900eb54b344bde389f98792e

Reply:
@joedragons we're going to archive this repo which disables issues